torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -510,6 +576,50 @@ def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
             model = self._annotate_for_static_quantization_config(model)
         return model
 
+    def _annotate_by_single_config(
+        self,
+        model: torch.fx.GraphModule,
+        quantization_config: Optional[QuantizationConfig],
+        filter_fn: Optional[Callable[[Node], bool]] = None,
+    ) -> torch.fx.GraphModule:
+        """Select the annotator functions according to the `quantization_config` and apply."""
+
+        # <TODO> implement the support for None to be canceling out previous annotations
+        if quantization_config is None:
+            return model
+
+        if (
+            quantization_config.input_activation
+            and quantization_config.input_activation.is_dynamic
+        ):
+            annotators = DYNAMIC_ANNOTATORS
+        else:
+            annotators = STATIC_ANNOTATORS
+            if quantization_config.is_qat:
+                # Apply QAT-specific annotators first
+                for annotator_func in STATIC_QAT_ONLY_ANNOTATORS.values():
+                    annotator_func(self, model, quantization_config, filter_fn)
+
+        for annotator_func in annotators.values():
```

Comment:
Sync with @yiliu30, For the operator in

- `propagation_quantizable_ops` like maxpool2d
- And `matmul` which is not in default recipe
 
we should also annotate it if it's inside the module of `module_name_qconfig`. Please add the support and also add it into document for this API `set_module_name_qconfig`.

Reply:
Thanks, added the support and documented it in `set_module_name_qconfig`.